### PR TITLE
VOH substraction feature added

### PR DESCRIPTION
**Pull request by vSebas**

### CODE REVIEW

File: usv_control/src/velocity_obstacle.cpp [hunk 1]
```diff
@@ -344,17 +344,57 @@ void reachable_velocities(){
...
+
   //Polygon_with_holes_2* ptr = &C_union;
   for(int i = 0; i<obstacle_list_.size(); ++i){
-    double obs_dist = sqrt(pow(obstacle_list_[i].x,2)+pow(obstacle_list_[i].y,2));
+    double obs_dist = sqrt(pow(obstacle_list_[i].x,2)+pow(obstacle_list_[i].y,2)) - obstacle_list_[i].r;
     // Codigo Sebas
-    
+    speed_th = obs_dist / time_horizon_;
+    obstacle_theta = atan2(obstacle_list_[i].x, obstacle_list_[i].y);
+    vel_th_x = speed_th * sin(obstacle_theta);
```
Comment (ivanacollg) [inline usv_control/src/velocity_obstacle.cpp]: Usar structs

File: usv_control/src/velocity_obstacle.cpp [hunk 2]
```diff
@@ -344,17 +344,57 @@ void reachable_velocities(){
...
+    intersect_l_y =  numerator / denominator;
+    // Intersect tan_r
+    numerator = (((obstacle_list_[i].tan_r.x*0)-(obstacle_list_[i].tan_r.y*0))*(vel_th_x-b)) - ((obstacle_list_[i].tan_r.x-0)*((vel_th_x*0)-(vel_th_y*b)));
+    denominator = ((obstacle_list_[i].tan_r.x-0)*(vel_th_y-0)) - ((obstacle_list_[i].tan_r.y-0)*(vel_th_x-b)); 
+    intersect_r_x =  numerator / denominator;
+    numerator = (((obstacle_list_[i].tan_r.x*0)-(obstacle_list_[i].tan_r.y*0))*(vel_th_y-0)) - ((obstacle_list_[i].tan_r.y-0)*((vel_th_x*0)-(vel_th_y*b)));
+    intersect_r_y =  numerator / denominator;
+
+    // Construct VOH
+    VOH.clear();
```
Comment (ivanacollg) [inline usv_control/src/velocity_obstacle.cpp]: Incluir vertices en Cono C, borrar VOH

**Review by ivanacollg — CHANGES_REQUESTED** [review]

File: usv_control/src/velocity_obstacle.cpp (hunk 2 above)
Comment (vSebas) [reply usv_control/src/velocity_obstacle.cpp]: Done

File: usv_control/src/velocity_obstacle.cpp (hunk 1 above)
Comment (vSebas) [reply usv_control/src/velocity_obstacle.cpp]: Done

**Pull request merged**